ios/make_ios_project.sh

```diff
@@ -251,14 +251,23 @@ fi
 # Clean up no-longer-needed electroncash/ dir that is outside of Xcode project
 rm -fr ${compact_name}/electroncash/*
 
-/usr/bin/env ruby update_project.rb
+# Can add this back when it works uniformly without issues 
+# /usr/bin/env ruby update_project.rb
 
 echo ''
 echo '**************************************************************************'
 echo '*                                                                        *'
 echo '*   Operation Complete. An Xcode project has been generated in "iOS/"    *'
 echo '*                                                                        *'
 echo '**************************************************************************'
+
+echo '  IMPORTANT!'
+echo '        Now you either need to manually add the library libxml2.tbd to the '
+echo '        project under "General -> Linked Frameworks and Libraries" or successfully '
+echo '        run the update_project.rb script to do it for you '
```

Comment:
I know this seems ridiculous but if you don't do it I will have to do it in a subsequent commit. Would you mind ending this sentence with a period `.`?  If you don't do it I will after merge.. thus adding 1 extra commit so.. hoping you don't mind doing it here to this PR. :)

Reply:
Eg: `run the update_project.rb script to do it for you.`  <-- period. :)

Reply:
OK. Added. No problem.
Also don't understand the differences.
Glad to help and see this merged.
I'm presuming you'll squash, but I can rebase/squash if needed